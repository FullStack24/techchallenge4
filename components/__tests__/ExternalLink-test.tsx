import * as React from 'react';
import renderer from 'react-test-renderer';
import { ExternalLink } from '../ExternalLink';

it('deve renderizar o ExternalLink corretamente', () => {
  const tree = renderer.create(
    <ExternalLink href="https://www.example.com" />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
