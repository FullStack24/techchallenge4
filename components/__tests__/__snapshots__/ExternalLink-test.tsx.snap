// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`deve renderizar o ExternalLink corretamente 1`] = `
<Text
  href="https://www.example.com"
  onPress={[Function]}
  role="link"
/>
`;
